docs/getting-started.md

```diff
@@ -41,5 +61,4 @@ import file from 'node_modules/garden/src/js/file.js';
 
 JavaScript components rely on jQuery to work, most of the time. Please note `garden.min.js` *does not* comes with jQuery. You have to add your own version of it.
 
-
 ##### Checkout the **<a href="https://github.com/leroy-merlin-br/garden" target="_blank" title="Garden Readme">README</a>** of the project in order how to build the project locally.
```

Comment:
It seems there is a word missing in this line. Maybe the sentence should be: `Checkout the ... of the project to know how to build the project locally.`